include/yaml-cpp/node/detail/node_data.h

```diff
@@ -114,7 +114,7 @@ class YAML_CPP_API node_data {
   mutable std::size_t m_seqSize;
 
   // map
-  typedef std::map<node*, node*> node_map;
+  typedef std::list< std::pair<node*, node*> > node_map;
```

Comment:
Make sure you run this change through clang-format. You shouldn't need the extra spaces around the < and >